### DISCUSSION TITLE
Consider building platform native versions of the Themes Playground

### DISCUSSION THREAD

**Discussion by ivirtex** [rydmike/flex_color_scheme#132] [Polls]
Hi @rydmike! Thank you for creating this awesome package, it makes theming so much easier in Flutter.

I don't know if this has been discussed yet, but as we know, current Flutter's performance on the web is not great, especially for an app like this, making it laggy and janky. The situation will probably improve with future WASM support, but I guess it's not coming to the next stable.

I think natively compiled versions of the Themes Playground would bring big performance gains and would make it more enjoyable to experiment with for now.

**Comment by rydmike** [comment]
Hi @ivirtex and thanks for using FlexColorScheme.

Providing and distribution prebuilt Windows and macOS executables is something that will be considered if/when the **Themes Playground** app is actually made to a "real" application and not just as a tutorial on how to make a dynamically themed app, which is how it started.

Some info about the Playground's roots here: https://docs.flexcolorscheme.com/playground#background

About executables, they are mentioned here as well as a possibility: https://docs.flexcolorscheme.com/playground#future for a future version.

All that said, the full **Themes Playground** app is included with the package repository, so if you clone it you can easily build Windows/macOS/Linux executables of it and use it instead of the Web version:
https://docs.flexcolorscheme.com/examples#building-the-examples

Hope this helps until there are prebuilt ones available for direct installation 😄

**Comment by rydmike** [comment]
If I have time to figure out how to automate making them with GitHub actions, I guess providing simple GitHub releases downloadable executables could be done earlier as well.

I'm going to convert this to a POLL in discussions to see if it gathers any more immediate interest. 

Hopefully SKIA builds will be back to offer better performance soon again too. At least on my MacBook 14" M1Pro the html build runs pretty well in a browser too, but yeah it is not stellar. 😄

**Comment by rydmike** [comment]
So much for making it a POLL. GitHub threw up a 404 when converting an issue to a discussion and selecting that it should become a POLL and I cannot find any way of editing and adding the POLL question. lol, well people can comment I guess or add thumbs up 👍🏻 😄
